module-4/backend/week_one.md

```diff
@@ -5,17 +5,60 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. What's the most useful thing you learned from completing the intermission week work?
+
+  I really enjoyed learning about non-relational databases with Neo4j and (FINALLY) implementing the Interactive Broker API. It was also very useful to get comfortable with writing python scripts without a web framework. I also took part of a datacamp course, which gave me a good understanding of how to create hacker data and interpret results.
+
 2. What are some tools to help debug JavaScript code?
+
+  - Developer tools (with many different functions)
+  - console.log(...)
+  - debugger
+
 3. What are some tools you need in order to unit test your JavaScript?
+
+  Mocha and chai, as well as decent enough compartmentalization of JavaScript code.
+
 4. What is the syntax for invoking a function?
+
+  invokeThisFunction() -> invokes
+
 5. What is CORS?
```

Comment:
These questions are from a previous version of the questions! Please pull down the latest version of the questions as specified in class. The future weeks will match up poorly without it